br/pkg/lightning/mydump/parser.go

```diff
@@ -303,20 +303,22 @@ func (parser *blockParser) readBlock() error {
 	}
 }
 
-var unescapeRegexp = regexp.MustCompile(`(?s)\\.`)
+var chunkParserUnescapeRegexp = regexp.MustCompile(`(?s)\\.`)
```

Comment:
Almost LGTM. I don't quite understand why this regexp has been preserved and will review it later.